Packages/VirtualMachine/Sources/VirtualMachineResourcesServiceEphemeral/VirtualMachineResourcesServiceEphemeral.swift

```diff
@@ -61,6 +61,7 @@ public struct VirtualMachineResourcesServiceEphemeral: VirtualMachineResourcesSe
     }
 
     public func createResourcesIfNeeded() async throws {
+        return;
```

Comment:
The code underneath this return statement will never be executed anymore. Is this a line left in by accident after debugging?

Reply:
Yes, this was left in by accident. I removed the return statement.